emails/views.py

```diff
@@ -326,7 +326,7 @@ def _sns_message(message_json):
             'update the forwarding settings in your dashboard.\n'
             '{extra_msg}---Begin Email---\n'
         ).format(
-            relay_address=display_email, extra_msg=attachment_not_supported
+            relay_address=to_address, extra_msg=attachment_not_supported
```

Comment:
What is this part fixing?

Reply:
broken Relay address on the text-only email client.
Broken:
```
This email was sent to your alias asd---<span>@</span>mail<span>.</span>dev<span>.</span>fxprivaterelay<span>.</span>nonprod<span>.</span>cloudops<span>.</span>mozgcp<span>.</span>net. To stop receiving emails sent to this alias, update the forwarding settings in your dashboard.
```
Fixed:
```
This email was sent to your alias asd---@mail.dev.fxprivaterelay.nonprod.cloudops.mozgcp.net. To stop receiving emails sent to this alias, update the forwarding settings in your dashboard.
```